tests/forgotten-password-view.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Can we rename the file to `forgotten-password-test-view.json` so it's clear from reading it that it isn't the view needed for the forgotten-password functionality?